modules/tracker/controllers/components/ApisubmissionComponent.php

```diff
@@ -185,8 +228,13 @@ public function post($args)
      * @path /tracker/submission/{id}
      * @http PUT
      * @param id
-     * @param uuid (Optional)
```

Comment:
Did you remove `uuid` as a param because you want it to be immutable through this API?  I think that is a worth an explicit comment somewhere in the function.

Also, the way the function creates a DAO from the args, `uuid` would be updated if passed.

    $submissionDao = $submissionModel->initDao('Submission', $args, $this->moduleName);